content/docs/gto/index.md

```diff
@@ -0,0 +1,39 @@
+# GTO Documentation
+
+**GTO** is a tool for creating an Artifact Registry in your Git repository. One
+of the special cases we would like to highlight is creating a
+[Machine Learning Model Registry](/doc/use-cases/model-registry).
+
+Such a registry serves as a centralized place to store and operationalize your
+artifacts along with their metadata; manage model life-cycle, versions &
```

Comment:
Are we talking about the general artifacts registry or specific models registry here? If the former,

```suggestion
Such a registry serves as a centralized place to store and operationalize your
artifacts along with their metadata; manage ML life-cycle, versions &
```